pyanaconda/bootloader.py

```diff
@@ -2303,16 +2418,27 @@ def install(self, args=None):
 
 
 # every platform that wants a bootloader needs to be in this dict
-bootloader_by_platform = {platform.X86: GRUB2,
-                          platform.EFI: EFIGRUB,
-                          platform.MacEFI: MacEFIGRUB,
-                          platform.PPC: GRUB2,
-                          platform.IPSeriesPPC: IPSeriesGRUB2,
-                          platform.NewWorldPPC: MacYaboot,
-                          platform.S390: ZIPL,
-                          platform.Aarch64EFI: Aarch64EFIGRUB,
-                          platform.ARM: EXTLINUX,
-                          platform.omapARM: EXTLINUX}
+if flags.cmdline.get("inst.legacygrub") == "1":
```

Comment:
Arguments are defined in pyanaconda/anaconda_argparse.py in that big pile of add_argument calls. The help output, from all of those help_parser.help_text, is defined in data/anaconda_options.txt (or you could use help=argparse.SUPPRESS to just hide it from --help). If you add a --legacygrub option there it will also get tied to inst.legacygrub on the command line